cpython-unix/targets.yml

```diff
@@ -562,7 +572,7 @@ x86_64-apple-darwin:
   target_cflags:
     - '-arch'
     - 'x86_64'
-    - '-mmacosx-version-min=10.9'
+    - '-mmacosx-version-min=10.15'
```

Comment:
I think this is fine, but we should also probably fix the upstream to use weak symbols here.

Reply:
See 

- https://github.com/python/cpython/issues/128134
- https://github.com/python/cpython/pull/127932

We could revert this for the next alpha, if we wanted.

This is currently unconditional and applies to other CPython versions.

I think it's fine regardless.